### ISSUE TITLE
METRIC DOC UPDATES

### ISSUE THREAD

**Issue by vivianyentran** [getsentry/sentry-docs#10188]
- [x] Metrics with CLI [NEW] ([PR](https://github.com/getsentry/sentry-docs/pull/10073))
- [x] Update [Trace View](https://docs.sentry.io/concepts/key-terms/tracing/trace-view) to include metrics
- [x] Update [Dashboards](https://docs.sentry.io/product/dashboards/) to include metrics ([PR](https://github.com/getsentry/sentry-docs/pull/10195))
- [x] Update [Metric Alerts](https://docs.sentry.io/product/alerts/create-alerts/metric-alert-config/) to include metrics ([PR](https://github.com/getsentry/sentry-docs/pull/10179))
- [x] Differentiate Metrics from custom performance metrics/measurements ([PR](https://github.com/getsentry/sentry-docs/pull/10143))
- [x] Add metrics to [What is Sentry](https://docs.sentry.io/product/) page
- [x] Update arcades & screenshots for metrics (remove beta note, reflect new left nav IA, part of explore)
```[tasklist]
### Tasks
- [ ] https://github.com/getsentry/sentry-docs/pull/10073
- [ ] https://github.com/getsentry/sentry-docs/pull/10179
- [ ] https://github.com/getsentry/sentry-docs/pull/10143
- [ ] https://github.com/getsentry/sentry-docs/pull/10195
```

**Comment by ale-cota** [comment]
hey, @vivianyentran! I looked at the [Trace View](https://docs.sentry.io/concepts/key-terms/tracing/trace-view) page in the docs and it didn't have any details about how metadata is displayed in the span details. There are many other things aside from custom metrics that appear there that are even more important than the custom metrics, so I felt it would be strange to include this. I opted for including the information on the Metrics Explorer page instead ([this PR](https://github.com/getsentry/sentry-docs/pull/10175) you already reviewed 🙏). Does that make sense to you too?
